COMETwebapp/Pages/_Host.cshtml

```diff
@@ -35,6 +35,7 @@
         <base href="~/" />
         <link href="COMETwebapp.styles.css" rel="stylesheet"/>
         <link href="css/app.css" rel="stylesheet"/>
+        <script src="Scripts/bootstrap.bundle.min.js"></script>
```

Comment:
Required for AntDesign?

Reply:
Not really, but enables the usage of some bootstrap functionalities (like [collapsible text](https://getbootstrap.com/docs/5.0/components/collapse/) used for error details in toast notification) besides the styles we already have